JitsiConferenceEvents.js

```diff
@@ -230,6 +230,15 @@ export const TRACK_MUTE_CHANGED = 'conference.trackMuteChanged';
 export const TRACK_REMOVED = 'conference.trackRemoved';
 
 /**
+ * Notifies for transcription status changes. The event provides the
+ * following parameters to its listeners:
+ *
+ * @param {String} status - The new status.
+ */
+export const TRANSCRIPTION_STATUS_CHANGED = 'xmpp.transcription_status_changed';
```

Comment:
Is there any reason why this isn't called "conference.transcriptionStatusChanged", the same way we call the other JitsiConferenceEvents?